### PR TITLE
Module 5 - React hooks

### PR DESCRIPTION
**Pull request by TashaUA**

### CODE REVIEW

File: src/components/MovieExtendedCard.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            <div className="movie-extended-card__info">
+                <p className="movie-extended-card__info-title">{title}
+                    <span className="movie-extended-card__info-rating">{vote_average}</span>
+                </p>
+                <p>{tagline}</p>
+                <p className="movie-extended-card__info-date"><span>{release_date.split('-')[0]}</span> {runtime} min
+                </p>
+                <p className="movie-extended-card__info-overview">{overview}</p>
+                {genres.join(", ")}
+                <p class="actions">
```
Comment (inductio) [inline src/components/MovieExtendedCard.jsx]: Should be `className`. Please fix.

File: src/containers/AddEditMovieDialog.jsx [hunk 2]
```diff
@@ -24,72 +23,59 @@ class AddEditMovie extends React.Component {
...
     };
 
-    render() {
-        return this.props.openEdit ? (
-                <Dialog onClose={this.props.onCloseDialog}
-                        title={this.state.isEdit ? 'Edit movie' : 'Add movie'}>
+        return props.openEdit && (
+                <Dialog onClose={props.onCloseDialog}
+                        title={isEdit ? 'Edit movie' : 'Add movie'}>
                     <form className="form form__add-movie">
```
Comment (inductio) [inline src/containers/AddEditMovieDialog.jsx:60]: Please, pay attention to warnings in console.
image
https://reactjs.org/docs/forms.html#controlled-components

Comment (inductio) [inline src/containers/AddEditMovieDialog.jsx:60]: Please, pay attention to warnings in console.
image
https://reactjs.org/docs/forms.html#controlled-components

File: src/containers/AddEditMovieDialog.jsx [hunk 3]
```diff
@@ -24,72 +23,59 @@ class AddEditMovie extends React.Component {
...
 
-    handleInput = (event) => {
+    const handleInput = (event) => {
         const {value, name} = event.target;
 
-        this.setState({
-            [name]: value
+        setMovie({
+            ...movie,
+            [event.target.name]: event.target.value
```
Comment (inductio) [inline src/containers/AddEditMovieDialog.jsx]: `[name]: value`
-  would be better here, variables are already assigned.

File: src/containers/MoviePage.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import MovieExtendedCard from "../components/MovieExtendedCard";
+import MoviesData from "../components/MoviesData";
+
+export default function MoviePage(props) {
+    const entry = MoviesData[8] || {};
```
Comment (inductio) [inline src/containers/MoviePage.jsx]: Please avoid magic number [8], would be better to assign it to sufficiently named variable (at least). 
Ideally to change logic and using dynamic values.

File: src/components/MovieExtendedCard.jsx (hunk 1 above)
Comment (TashaUA) [reply src/components/MovieExtendedCard.jsx]: thanks fixed

File: src/containers/AddEditMovieDialog.jsx (hunk 2 above)
Comment (TashaUA) [reply src/containers/AddEditMovieDialog.jsx:60]: fixed

File: src/containers/AddEditMovieDialog.jsx (hunk 3 above)
Comment (TashaUA) [reply src/containers/AddEditMovieDialog.jsx]: fixed

File: src/containers/MoviePage.jsx (hunk 4 above)
Comment (TashaUA) [reply src/containers/MoviePage.jsx]: also fixed

**Pull request merged**